sorc/chgres_cube.fd/atmosphere.F90

```diff
@@ -343,8 +372,8 @@ subroutine atmosphere_driver(localpet)
  method=ESMF_REGRIDMETHOD_BILINEAR
 
  print*,"- CALL FieldRegridStore FOR 3D-WIND WEST EDGE."
- call ESMF_FieldRegridStore(wind_target_grid, &
-                            wind_w_target_grid, &
+ call ESMF_FieldRegridStore(xwind_target_grid, &
```

Comment:
Looks like the print statement should be updated here.

Reply:
Done